cfme/tests/cloud_infra_common/test_power_control_rest.py

```diff
@@ -43,44 +43,138 @@ def verify_vm_power_state(vm, state, minutes=10, action=None):
         message='Wait for VM to {} (current state: {})'.format(state, vm.power_state))
 
 
+def get_collection(vm, rest_api):
+    if vm.cloud:
+        return rest_api.collections.instances
+    elif vm.cloud is False:
+        return rest_api.collections.vms
+    return None
+
+
 @pytest.mark.uncollectif(
-    lambda provider: provider.category == 'cloud' and current_version() <= "5.5.2.4")
+    lambda provider: provider.category == 'cloud' and current_version() < "5.6.0")
 @pytest.mark.parametrize("from_detail", [True, False], ids=["cfrom_detail", "from_collection"])
 def test_stop(rest_api, vm_obj, from_detail):
+    """Test stop of vm
+
+    Prerequisities:
+
+        * An appliance with ``/api`` available.
+        * VM
+
+    Steps:
+
+        * POST /api/vms/<id> (method ``stop``)
+        OR
+        * POST /api/vms (method ``stop``) with ``href`` of the vm or vms
+
+    Metadata:
+        test_flag: rest
+    """
     vm = vm_obj.get_vm_via_rest()
     assert "stop" in vm.action
     verify_vm_power_state(vm, state=vm_obj.STATE_ON)
     if from_detail:
         vm.action.stop()
     else:
         rest_api.collections.vms.action.stop(vm)
-    wait_for(lambda: vm .power_state == vm_obj.STATE_OFF,
+    wait_for(lambda: vm.power_state == vm_obj.STATE_OFF,
         num_sec=1000, delay=20, fail_func=vm.reload,
         message='Wait for VM to stop (current state: {})'.format(vm.power_state))
 
 
 @pytest.mark.uncollectif(
-    lambda provider: provider.category == 'cloud' and current_version() <= "5.5.2.4")
+    lambda provider: provider.category == 'cloud' and current_version() < "5.6.0")
```

Comment:
Just clarifying, this will drop all testing for cloud providers less than 5.6.0, is this due to incompatibility? We can't do cloud start operation on lower than 5.6?